src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -0,0 +1,131 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    private static final String INSERT_DRIVER_REQUEST
+            = "INSERT INTO drivers(name, licence_number) values(?,?);";
+    private static final String GET_ALL_DRIVERS_REQUEST
+            = "SELECT * FROM drivers WHERE is_deleted = false";
+    private static final String UPDATE_DRIVER_REQUEST
+            = "UPDATE drivers SET name=?,licence_number=? WHERE id=?";
+    private static final String DELETE_DRIVER_REQUEST
+            = "UPDATE drivers SET is_deleted = true WHERE id = ?";
+    private static final String GET_DRIVER_REQUEST
+            = "SELECT * FROM drivers WHERE is_deleted = false AND id = ?";
+    private static final String CANT_GET_ALL_MESSAGE = "Can't get all drivers from DB.";
+    private static final String CANT_CREATE_MESSAGE = "Can't insert driver to DB.";
+    private static final String CANT_UPDATE_MESSAGE = "Can't update driver in DB.";
+    private static final String CANT_DELETE_MESSAGE = "Can't delete driver from DB.";
+    private static final String CANT_GET_MESSAGE = "Can't get driver by id from DB.";
+    private static final String NAME = "name";
+    private static final String LICENCE_NUMBER = "licence_number";
+    private static final String ID = "id";
+
+    @Override
+    public List<Driver> getAll() {
+        List<Driver> driverList = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                Statement getAllDriversStatement = connection.createStatement();) {
+            ResultSet resultSet = getAllDriversStatement
+                    .executeQuery(GET_ALL_DRIVERS_REQUEST);
+            while (resultSet.next()) {
+                String name = resultSet.getString(NAME);
+                String licenceNumber = resultSet.getString(LICENCE_NUMBER);
+                Long id = resultSet.getObject(ID, Long.class);
+                Driver driver = new Driver();
+                driver.setId(id);
+                driver.setName(name);
+                driver.setLicenseNumber(licenceNumber);
+                driverList.add(driver);
```

Comment:
it's better to create a separate method to work with ResultSet. it should take ResultSet as a parameter and return Driver

Reply:
fixed